### DISCUSSION TITLE
Unable to connect to the server: x509: certificate is valid for

### DISCUSSION THREAD

**Discussion by sixwaaaay** [k3s-io/k3s#5241] [Q&A]
the response is: Unable to connect to the server: x509: certificate is valid for

**Comment by brandond** [comment]
Can you post the rest of the error message, as well as some information on what you're trying to do and what happens?

**Comment by sixwaaaay** [comment]
when I use kubectl locally to connect remote k3s machine, it report this

**Reply by brandond** [reply]
It sounds like you might need to use the --tls-san flag? 
https://rancher.com/docs/k3s/latest/en/installation/install-options/server-config/#listeners
